torchgen/gen.py

```diff
@@ -1247,7 +1252,10 @@ def compute_registration_declarations(
         # TODO: What exactly is the semantics of the 'dispatch' field?
         "dispatch": str(
             {k for k, v in backend_indices.items() if v.has_kernel(f)}
-            != {DispatchKey.CompositeImplicitAutograd}
```

Comment:
This looks like it'll change the output of "dispatch" for existing native functions that only have a `CompositeImplicitAutograd` entry.

Feels kinda ugly, but I guess you want to check for `... != {CompositeImplicitAutograd} and ... != {CompositeImplicitAutograd, CompositeImplicitAUtogradNestedTensor}`